src/connection/sqlJob.ts

```diff
@@ -4,6 +4,7 @@ import { ServerComponent } from "./serverComponent";
 import { JDBCOptions, ConnectionResult, Rows, QueryResult, JobLogEntry, CLCommandResult, VersionCheckResult, GetTraceDataResult, ServerTraceDest, ServerTraceLevel, SetConfigResult, QueryOptions } from "./types";
 import { Query } from "./query";
 import { EventEmitter } from "stream";
+import { commands } from "vscode";
```

Comment:
`vscode` should not be imported in this class as we will make it reusable outside of VS Code. Sorry!

Reply:
This is still requred ^